Source/santad/SNTExecutionController.mm

```diff
@@ -63,6 +89,9 @@ @interface SNTExecutionController ()
 
 @implementation SNTExecutionController {
   std::shared_ptr<TTYWriter> _ttyWriter;
+  absl::Mutex _filterMutex;
```

Comment:
We should probably name this `_entitlementFilterMutex`. While we're not filtering anything else today if we we'll need to deconflict the name later.